federation/cluster/federation-up.sh

```diff
@@ -90,7 +90,7 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
+      --apiserver-arg-overrides="--storage-backend=etcd2,--runtime-config=api/all=true" \
```

Comment:
We are in the process of transitioning to kubefed for provisioning the federation control plane. Could you add the same vlaue flag here as well. https://github.com/kubernetes/kubernetes/blob/95ce463e95e5d4f1286b509e0de0518e6214e3c3/federation/pkg/kubefed/init/init.go#L678-L685